sdcm/nemesis.py

```diff
@@ -2800,6 +2802,57 @@ def _verify_cdc_feature_status(self, keyspace: str, table: str, cdc_settings: di
             assert actual_cdc_settings == cdc_settings, \
                 f"CDC extension settings are differs. Current: {actual_cdc_settings} expected: {cdc_settings}"
 
+    def _add_new_node_in_new_dc(self) -> BaseNode:
+        new_node = self.cluster.add_nodes(1, dc_idx=0, enable_auto_bootstrap=True)[0]  # add node
+        with new_node.remote_scylla_yaml() as scylla_yml:
+            scylla_yml.rpc_address = new_node.ip_address
+            scylla_yml.seed_provider = [SeedProvider(class_name='org.apache.cassandra.locator.SimpleSeedProvider',
+                                                     parameters=[{"seeds": self.tester.db_cluster.seed_nodes_ips}])]
+        new_node.remoter.run("sudo sh -c 'echo dc_suffix=_nemesis_dc >> /etc/scylla/cassandra-rackdc.properties'")
+        self.cluster.wait_for_init(node_list=[new_node], timeout=300,
```

Comment:
you are right, in the function called `_add_and_init_new_cluster_node()` we wait `timeout=HOUR_IN_SEC * 6`

Reply:
As I understand, in typical add node to cluster test, when a node is in the same DC, with simple replication strategy, data is always getting distributed to new node - in long tests it may take quite a lot. 
Here, when we have new DC, only keyspaces with NetworkTopology strategy (with rf>0 for new DC) will distribute data to new DC.
In the test, only system keyspaces are getting NetworkTopology strategy - and I presume these tables don't contain much data.
I may modify all existing kesypaces to use NetworkTopology to induce more real case scenario. I'm not sure yet how c-s will react when replication strategy changes during stress time.

Reply:
Not accurate, because this nemesis may run as part of multi-dc longevity, where you already have NetworkTopology set for few keyspaces.